pkgs/os-specific/linux/sysdig/default.nix

```diff
@@ -24,6 +24,7 @@ stdenv.mkDerivation rec {
   cmakeFlags = [
     "-DUSE_BUNDLED_DEPS=OFF"
     "-DSYSDIG_VERSION=${version}"
+    "-DCREATE_TEST_TARGETS=OFF"
```

Comment:
does it disable tests ?

Reply:
The tests were not executed before and this is also not feasible since we cannot load the sysdig kernel module.